onnxruntime/test/optimizer/transpose_optimizer_test.cc

```diff
@@ -2969,11 +2969,23 @@ TEST(TransposeOptimizerTests, TestBroadcastOpsDiv) {
     EXPECT_EQ(transpose_cost, 0);
   };
 
+#if defined(_M_ARM64) && _MSC_VER >= 1930
+  // Slight difference in Windows ARM64 VS 2022:
+  // expected 19.3678 (419af143), got 19.3678 (419af144), diff: 1.90735e-06, tol=0
+  TransformerTester(build_test_case_1,
+                    check_optimized_graph_1,
+                    TransformerLevel::Default,
+                    TransformerLevel::Level1,
+                    /*opset_version*/ {15, 18},
+                    /*per_sample_tolerance*/ 1e-07,
+                    /*relative_per_sample_tolerance*/ 1e-06);
```

Comment:
Tolerance is computed as follows:
`tol = per_sample_tolerance + relative_per_sample_tolerance * expected_value`